server/src/inmantals/pipeserver.py

```diff
@@ -38,12 +38,12 @@ def main():
     stdin = PipeIOStream(sys.stdin.fileno())
     stdout = PipeIOStream(sys.stdout.fileno())
     handler = InmantaLSHandler(stdin, stdout, "0.0.0.0")
-    sys.stderr.write("starting")
+    sys.stderr.write("starting\n")
```

Comment:
Write these to stdout instead? The client now logs stdout as well.

Reply:
I think that's only true for the `tcpserver`

Reply:
Right, nevermind then.